### PR TITLE
Fix issue when root disk not present

### PR DESCRIPTION
**Pull request by presztak**
This PR addresses the following issues:
- Properly handle situations where the root disk device is not present.
- Properly handle situations where the root disk device name is different from 'root'.

Fixes: #10

### CODE REVIEW

File: src/pages/instances/InstanceOverviewMetrics.tsx [hunk 1]
```diff
@@ -28,6 +28,28 @@ const InstanceOverviewMetrics: FC<Props> = ({ instance, onFailure }) => {
...
+
+    return Object.keys(state.disk).length > 0;
+  };
+
+  const getRootDisk = (state: LxdInstanceState) => {
+    if (!hasRootDisk(state)) {
+      return null;
+    }
+
+    return Object.values(state.disk)[0];
```
Comment (stgraber) [inline src/pages/instances/InstanceOverviewMetrics.tsx:48]: What happens if you have multiple disks attached and have one show up on the list just before the root disk?

Do we know that [0] will always be the root disk?

Comment (presztak) [reply src/pages/instances/InstanceOverviewMetrics.tsx:48]: So, for vms we always have only root disk inside of stats: https://github.com/lxc/incus/blob/e8aa69d75bf82ab3f7c9098c7b05704d4492066a/internal/server/instance/drivers/driver_qemu.go#L8094-L8098

but for containers, you are right, we can have multiple disks: https://github.com/lxc/incus/blob/e8aa69d75bf82ab3f7c9098c7b05704d4492066a/internal/server/instance/drivers/driver_lxc.go#L7507, but there is no way to distinguish them as struct looks like:

```
{
  dev1: {
    total: 100,
    usage: 10,
  },
  dev2: {
    total: 100,
    usage: 10,
  },
}
```

Comment (stgraber) [reply src/pages/instances/InstanceOverviewMetrics.tsx:48]: Yeah, you need to look at expanded_devices on the instance config, then get the root disk device name from that by looking for whatever is `type==disk and path==/`. Once you know the name of the device, then you can get it from the state.

Comment (presztak) [reply src/pages/instances/InstanceOverviewMetrics.tsx]: Fixed.

**Review by stgraber — APPROVED** [review]

**Pull request merged**